### PR TITLE
Custom form elements

### PR DESCRIPTION
**Pull request by xavijam**
Here we go!

screen shot 2016-02-15 at 19 37 06
screen shot 2016-02-15 at 19 37 09

CR: @javierarce 
cc @piensaenpixel @viddo

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/components/form-components/field.tpl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="u-tSpace-xl CDB-Text">
+  <p class="CDB-Legend u-upperCase u-iBlock CDB-Text is-semibold CDB-Size-small u-rSpace--m" for="<%= editorId %>">
+    <%- title %>
+  </p>
+  <div class="CDB-Text CDB-Size-medium u-iBlock">
+    <span data-editor></span>
+    <div data-error></div>
+    <div><%= help %></div>
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/components/form-components/field.tpl]: Was this (`<%=`) written with the equal sign in purpose?

**Comment by javierarce** [comment]
:ok_hand:

**Comment by xavijam** [comment]
retest this please

File: lib/assets/javascripts/cartodb3/components/form-components/field.tpl (hunk 1 above)
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/form-components/field.tpl]: Nah, it was copied from what backbone-forms has.

**Comment by xavijam** [comment]
Frontend tests passed, merging @javierarce :).

**Pull request merged**